Mobile/starter_project_matador/lib/features/profile page/presentation/widget/posts_widget.dart

```diff
@@ -0,0 +1,214 @@
+                  SizedBox(width: 30),
+                  const Icon(
+                    Icons.toc,
+                    color: Colors.grey,
+                    size: 30,
+                  ),
+                ],
+              ),
+            ),
+            Row(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(30, 10, 10, 0),
+                  child: Container(
+                    width: 100,
+                    height: 120,
+                    decoration: const BoxDecoration(
+                        image: DecorationImage(
+                          image: AssetImage('../assets/laser.jpg'),
+                          fit: BoxFit.cover,
+                        ),
+                        borderRadius: BorderRadius.all(Radius.circular(20))),
+                  ),
+                ),
+                SizedBox(width: screenWidth.size.width * 0.05),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(0, 0, 0, 10),
+                      child: Text(
+                        'BIG DATA                                  ',
+                        style: TextStyle(
+                            fontSize: 17, color: Colors.blue.withOpacity(0.6)),
+                      ),
+                    ),
+                    const Padding(
+                      padding: EdgeInsets.fromLTRB(0, 0, 0, 20),
+                      child: Text(
+                        'Why Big Data Needs Thick Data?',
+                        style: TextStyle(
+                            fontSize: 15, fontWeight: FontWeight.w600),
+                      ),
+                    ),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        Row(children: const [
+                          Icon(
+                            Icons.thumb_up,
+                            color: Colors.grey,
+                          ),
+                          Text('2.1k')
+                        ]),
+                        SizedBox(width: 23),
+                        Row(
+                          children: const [
+                            Icon(Icons.schedule),
+                            Text('1hr ago')
+                          ],
+                        ),
+                        SizedBox(width: 23),
+                        const Icon(
+                          Icons.bookmark,
+                          color: Colors.blue,
+                        ),
+                        const Text('        ')
+                      ],
+                    )
+                  ],
+                )
+              ],
+            )
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+//The Second card in my posts section
+
+class Second_post extends StatefulWidget {
```

Comment:
First post and Second post are not needed you can build it with a list view and use one widget. you can name the widget PostCard or something else more descriptive

Reply:
Its modularized into one class now.